### ISSUE TITLE
3XTF32 issue with the most recent cuquantum

### ISSUE THREAD

**Issue by sss441803** [NVIDIA/cuQuantum#129]
When I create a `Network` using
```python
Network(eq, *operands, options={'compute_type': cuquantum.cutensornet.ComputeType.COMPUTE_3XTF32})
```
it gives me the following error:
```console
ValueError: <ComputeType.COMPUTE_3XTF32: 8192> is not a valid ComputeType
```
This is not a problem with `TF32`, however. This is because there is no `3XTF32` in `cuQuantum/python/cuquantum
/_utils.pyx`?
I have
```
cuda-version              11.8                 h70ddcb2_3    conda-forge
cudatoolkit               11.8.0              h4ba93d1_13    conda-forge
cupy                      13.0.0          py311h878bca4_3    conda-forge
cupy-core                 13.0.0          py311heecd119_3    conda-forge
cuquantum-python          24.03.0         py311h8bf0e4b_3    conda-forge
custatevec                1.6.0                h56904bc_3    conda-forge
cutensor                  2.0.1.2              hcdd5f01_0    conda-forge
cutensornet               2.4.0           nompi_h56904bc_103    conda-forge
```

**Comment by yangcal** [comment]
Thanks for reporting. We're aware of the issue and are working on a fix.

**Comment by yangcal** [comment]
This has been fixed in the latest cuquantum-python v24.08. Please check out our new release.
